app/orm-adapters/objection.ts

```diff
@@ -189,10 +195,12 @@ export default class ObjectionAdapter extends ORMAdapter {
             return super.$parseDatabaseJson(json);
           }
         }
+
         Object.defineProperty(ObjectionModel, 'name', {
-          value: startCase(type) + 'ObjectionModel'
+          value: `${chain(type).startCase().replace(' ', '').value()}ObjectionModel`
```

Comment:
Using `chain()` here seems to just add verbosity, since we are only using a single lodash method.